tests/config/pytest_modules.yml

```diff
@@ -3948,6 +3952,10 @@ subworkflows/fastq_subsample_fq_salmon:
   - subworkflows/nf-core/fastq_subsample_fq_salmon/**
   - tests/subworkflows/nf-core/fastq_subsample_fq_salmon/**
 
+subworkflows/gatk_tumor_only_somatic_variant_calling:
```

Comment:
This should not be here, the subworkflow has been renamed.

Reply:
Yeah you should be fine to remove it